<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotificationsDialog</class>
 <widget class="QDialog" name="NotificationsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="serverBox">
     <property name="title">
      <string>Server Settings</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="serverLabel">
          <property name="text">
           <string>Server</string>
          </property>
          <property name="buddy">
           <cstring>serverEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="serverEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="portLabel">
          <property name="text">
           <string>Port</string>
          </property>
          <property name="buddy">
           <cstring>portSpin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="portSpin">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
          <property name="value">
           <number>25</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="loginLabel">
          <property name="text">
           <string>Login</string>
          </property>
          <property name="buddy">
           <cstring>loginEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="loginEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="pwdLabel">
          <property name="text">
           <string>Pwd</string>
          </property>
          <property name="buddy">
           <cstring>passEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="fromLabel">
          <property name="text">
           <string>From</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fromEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="tlsCheck">
          <property name="text">
           <string>TLS</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="secureCheck">
          <property name="text">
           <string>Secure</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="importantBox">
     <property name="title">
      <string>Important Messages Only</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QTextEdit" name="importantEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="allBox">
     <property name="title">
      <string>All Messages</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QTextEdit" name="allEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="eventBox">
     <property name="title">
      <string>Events</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QCheckBox" name="scanCheck">
        <property name="text">
         <string>End of Scan</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Error Messages</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NotificationsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NotificationsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>636</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eventBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>scanCheck</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eventBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>587</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
